assets/js/core/WakaTimeCore.js

```diff
@@ -183,33 +225,23 @@ class WakaTimeCore {
      * @param debug
      */
     sendHeartbeat(entity, debug) {
```

Comment:
`entity` is another name for `url` or `domain`, but now it's a heartbeat object containing a url so lets rename the `entity` arg to `heartbeat`.